styles.css

```diff
@@ -9,14 +9,14 @@
 }
 
 html {
-	overflow: hidden;
+  height: 100%;
+  overflow: hidden;
 }
-
-html,
 body {
-	height: 100%;
-	margin: 0;
-	padding: 0;
+  margin:0;
+  padding:0;
+  font-family: sans-serif;
```

Comment:
If you want to affect only the font of the web component you need to put that inside it